### DISCUSSION TITLE
notification_receive event for iOS

### DISCUSSION THREAD

**Discussion by NorbertSanpruch** [invertase/react-native-firebase#6973] [Q&A]
`notification_receive` is one of the predefined events by react native firebase analytics, but it only works on android. Can I somehow achieve notification receive event on iOS?

**Comment by mikehardy** [comment]
Can you point to the code where notification_receive is defined + sent as an analytics event in react-native-firebase?
I'm not sure it exists here.
I think what may be happening is that this is done automatically - *under* react-native-firebase - by the firebase-android-sdk library
I'm not sure how the firebase-ios-sdk library handles it for iOS, but if my assumption  / assertion that the underlying SDKs is doing it is correct, then it may just be a platform difference in the SDKs
